tests/test_xgcm_util.py

```diff
@@ -0,0 +1,13 @@
+import xarray as xr
+import xgcm
+
+import pop_tools
+from pop_tools import DATASETS
+
+
+def test_xgcm_grid():
+    fname = DATASETS.fetch('tend_zint_100m_Fe.nc')
+    ds = xr.open_dataset(fname)
+    grid = pop_tools.get_xgcm_grid(ds, metrics=None)
+    assert isinstance(grid, xgcm.Grid)
+    assert set(['X', 'Y', 'Z']) == set(grid.axes.keys())
```

Comment:
need to test that `ds` has renamed dimensions.

Reply:
Here's a file: `/glade/p/cgd/oce/projects/JRA55/IAF/g.e20.G.TL319_t13.control.001_hfreq/ocn/hist/g.e20.G.TL319_t13.control.001_hfreq.pop.h.0049-12.nc`

The variables `UEU`,  `VNU` should be tested. These have 3 in the second and third position of `grid_loc`